### PR TITLE
update core validator docs to include new validators

### PR DESCRIPTION
**Pull request by olensmar**
This PR...

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [X] updated the docs
- [ ] added a test

### CODE REVIEW

File: packages/validation/README.md [hunk 1]
```diff
@@ -24,10 +24,16 @@ Monokle Validation is a TypeScript library to validate your Kubernetes resources
...
-- Open Policy agent validates security policies to reduce your attack surface.
+The validation engine comes with a number of core plugins to provide you with comprehensive
+validation possibilities for K8s configurations out of the box:
+
+- **Pod Security Standards** validation for secure deployments
+- **Kubernetes Schema** validation to ensure your resource are compliant with their schemas and a target K8s version
+- **Resource links** validates that reference to other Kubernetes resources are valid.
+- **Metadata** validation for standard and custom labels/annotations
+- **Common practices** validation for basic configuration sanity
+- **AquaSec security policies** based on OPA (Open Policy Agent) to reduce your attack surface.
```
Comment (WitoDelnat) [inline packages/validation/README.md:35]: I think you also need to update a display name change of the OPA plugin?

Comment (olensmar) [reply packages/validation/README.md:35]: you're right - need to make sure we can actually use their name.. perhaps remove AquaSec for now and just use "Security Policies" !?

File: packages/validation/docs/core-plugins.md [hunk 2]
```diff
@@ -1,80 +1,111 @@
 ## Core Validation plugins
 
-There are four cores plugins:
+There are several cores plugins:
 
-- [Open Policy agent](#open-policy-agent)
-- [Kubernetes Schema](#kubernetes-schema)
+- [Kubernetes Schema](#kubernetes-schema) - includes deprecation/violation
```
Comment (f1ames) [inline packages/validation/docs/core-plugins.md:5]: Maybe here, we can duplicate plugins descriptions from https://github.com/kubeshop/monokle-core/pull/409/files#diff-d5870eae7fbe90f181dbda208cc45765807cebe168d9b7cbf2cf3d497e28960cR30-R36?

**Review by f1ames — DISMISSED** [review]
Somehow missed review request, sorry for that. Just one suggestion from me, apart from that looks good 👍

**Comment by olensmar** [comment]
thanks @WitoDelnat @f1ames - i've updated accordingly (I think...)

**Review by f1ames — APPROVED** [review]
👍

**Pull request merged**